ql/src/java/org/apache/hadoop/hive/ql/ddl/database/desc/DescDatabaseFormatter.java

```diff
@@ -69,6 +71,12 @@ void showDatabaseDescription(DataOutputStream out, String database, String comme
       if (ownerType != null) {
         builder.put("ownerType", ownerType.name());
       }
+      if (null != connectorName) {
```

Comment:
Should there be a field that says whether the database is native or remote?

Reply:
the other style is old code that already existed where as I added 2 clauses with different style. I will make them consistent.